slsim/lens_pop.py

```diff
@@ -137,6 +140,24 @@ def __init__(
                 sky_area=sky_area,
             )
 
+        elif deflector_type == "halo-models":
+            from slsim.Deflectors.compound_lens_halos_galaxies import (
```

Comment:
is there a way to test a simple LensPop() instance with SLHammock? Just to make sure the integration works

Reply:
For example, I think you can run the following commands to test a simple LensPop() instance with "halo-models."

from astropy.cosmology import FlatLambdaCDM
from astropy.units import Quantity
from slsim.lens_pop import LensPop

cosmo = FlatLambdaCDM(H0=70, Om0=0.3, Ob0 = 0.05)
sky_area = Quantity(value=0.001, unit="deg2")

kwargs_deflector_cut = None
kwargs_source_cut = {"band": "g", "band_max": 28, "z_min": 0.1, "z_max": 5.0}

skypy_config = "/path/to/data/SkyPy/lsst-like.yml"
slhammocks_config = "/path/to/data/SL-Hammocks/pop_salp_a0001_zl001_to_5_wo_sub.csv"
gg_lens_pop = LensPop(
    deflector_type="halo-models",
    source_type="galaxies",
    kwargs_deflector_cut=kwargs_deflector_cut,
    kwargs_source_cut=kwargs_source_cut,
    kwargs_mass2light=None,
    skypy_config=skypy_config,
    slhammocks_config=slhammocks_config,
    sky_area=sky_area,
    cosmo=cosmo,
)
gg_lens_pop._lens_galaxies.draw_deflector()